src/test/java/org/folio/edge/patron/utils/PatronMockOkapi.java

```diff
@@ -184,6 +193,21 @@ public void getAccountHandler(RoutingContext ctx) {
         .setStatusCode(404)
         .putHeader(HttpHeaders.CONTENT_TYPE, TEXT_PLAIN)
         .end(patronId + " not found");
+    } else if(-1 == offset) {
+      ctx.response()
+        .setStatusCode(400)
+        .putHeader(HttpHeaders.CONTENT_TYPE, TEXT_PLAIN)
+        .end(wrongOffsetMessage);
```

Comment:
If you haven't already done so, please double check that this reflects how mod-patron acts when offset is < 0

Reply:
I check, here is the response from mod-patron: 
![image](https://user-images.githubusercontent.com/60778653/145995872-526d7e00-b4ec-4073-9570-fbf13fcaf96c.png)